docs/cloud/managing-airbyte-cloud/dbt-cloud-integration.md

```diff
@@ -4,6 +4,12 @@ products: cloud
 
 # Use the dbt Cloud integration 
 
+:::warning
+Normalization and Custom Transformation are deprecated features. 
+Destinations using Normalization will be replaced by [Typing and Deduping](/using-airbyte/core-concepts/typing-deduping.md).
+Custom Transformation will be removed on March 31. For more information, visit [here](https://github.com/airbytehq/airbyte/discussions/34860).
```

Comment:
To be explicit, dbt cloud will remain available as an option for airbyte cloud. At least for the short-term future.